index.yaml

```diff
@@ -3922,6 +3922,31 @@ entries:
     urls:
     - assets/rancher-vsphere-csi/rancher-vsphere-csi-2.3.0.tgz
```

Comment:
I wonder why do we release 2.1.1 if 2.3.0 already exists?

Reply:
This is answered by https://github.com/rancher/charts/pull/1598#discussion_r743270395